pkgs/by-name/pi/pixelpwnr-server/package.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, cmake, freetype, libX11
+, libXcursor, libXrandr, libXi, libGL, pkg-config }:
+
+rustPlatform.buildRustPackage {
+  pname = "pixelpwnr-server";
+  version = "unstable-2024-03-13";
```

According to the [nixpkgs' package naming conventions](https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming) the version should be as follows:

```suggestion
  version = "0-unstable-2024-03-13";
```